recipes/shared-memory-dict/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  number: 0
+
+outputs:
+  - name: shared-memory-dict
+    build:
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<4.0
+      run:
+        - python >=3.6,<4.0
+    test:
+      imports:
+        - shared_memory_dict
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: shared-memory-dict-aiocache
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('shared-memory-dict', exact=True) }}
+        - aiocache >=0.11.1,<0.12.0
+    test:
+      imports:
+        - shared_memory_dict
+
+  - name: shared-memory-dict-all
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('shared-memory-dict', exact=True) }}
+        - aiocache >=0.11.1,<0.12.0
+        - django >=3.0.8,<4.0.0
```

Comment:
To make your life easier, you can also just install `shared-memory-dict-aiocache` and `shared-memory-dict-django` here. Then you only need to update dependencies once.